index.html

```diff
@@ -1070,8 +1262,13 @@ <h3>Role Mapping Table</h3>
 								</td>
 								<td><p>Expose <code>ROLE_SECTION</code> and object attribute
 											<code>xml-roles:doc-pullquote</code>.</p></td>
-								<td>AXRole: <code>AXGroup</code><br /> AXSubrole: <code>AXApplicationGroup</code><br />
-									AXRoleDescription: <code>'group'</code>
+								<td>
+									<ul>
+										<li>AXRole: <code>AXGroup</code></li>
+										<li>AXSubrole: <code>AXApplicationGroup</code></li>
+										<li>AXRoleDescription: <code>'group'</code></li>
+										<li>AXCustomContent:<code>{ label: "type", value: "pull quote" }</code></li>
```

Comment:
TBD… "quotation" rather than "pull quote"?

Reply:
I worry that it could confuse readers to call pull quotes simple quotations. The meaning and purpose is lost.

Reply:
okay